CoffeeChat/Controller/EditLocationAvailabilityViewController.swift

```diff
@@ -0,0 +1,296 @@
+//
+//  EditLocationAvailabilityViewController.swift
+//  CoffeeChat
+//
+//  Created by Amy Chin Siu Huang on 11/19/20.
+//  Copyright © 2020 cuappdev. All rights reserved.
+//
+
+import UIKit
+
+// MARK: - Location UICollectionView Header
+private class LocationHeaderLabel: UICollectionReusableView {
+
+    private let label = UILabel()
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        label.font = ._16CircularStdBook
+        addSubview(label)
+        label.snp.makeConstraints { make in
```

Comment:
Can you add a line break between `addSubview(label)` and `label.snp.makeConstraints`?